### ISSUE TITLE
"layers not yet deleted!" error while rendering

### ISSUE THREAD

**Issue by thekalinga** [synfig/synfig#571]
**Synfig version & platform**:
synfig 1.3.8
Ubuntu 14.04

**Issue description**:

I am trying to render the project I'm working on usign the following command
```
synfig -q \
--benchmarks \
--verbose 10 \
--input-file /home/user/Desktop/Presentation/Request-Per-Thread-End-To-End-Flow.sif \
--target ffmpeg \
--video-codec libx264 \
--video-bitrate 2000 \
--quality 0 \
--threads 4 \
--start-time 0 \
--end-time 45 \
--output-file /home/user/Desktop/Presentation/Request-Per-Thread-End-To-End-Flow.mkv &> synfig-rendering-error.log
```
and synfig fails with the following error

```
/home/user/Desktop/Presentation/Request-Per-Thread-End-To-End-Flow.sif: Rendered in 211.124 seconds.
Done.
frame=   46 fps=0.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   46 fps=0.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   46 fps=0.2 q=-1.0 Lsize=      35kB time=00:00:01.79 bitrate= 161.2kbits/s speed=0.0087x    
video:34kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 3.019072%
[libx264 @ 0x1aceac0] frame I:1     Avg QP:21.51  size: 10638
[libx264 @ 0x1aceac0] frame P:12    Avg QP: 5.63  size:  1684
[libx264 @ 0x1aceac0] frame B:33    Avg QP: 4.24  size:   107
[libx264 @ 0x1aceac0] consecutive B-frames:  4.3%  0.0%  0.0% 95.7%
[libx264 @ 0x1aceac0] mb I  I16..4:  0.1% 94.5%  5.3%
[libx264 @ 0x1aceac0] mb P  I16..4:  0.0%  0.0%  0.0%  P16..4:  1.9%  0.0%  0.0%  0.0%  0.0%    skip:98.0%
[libx264 @ 0x1aceac0] mb B  I16..4:  0.0%  0.0%  0.0%  B16..8:  0.6%  0.0%  0.0%  direct: 0.0%  skip:99.4%  L0:10.8% L1:89.1% BI: 0.1%
[libx264 @ 0x1aceac0] final ratefactor: 0.13
[libx264 @ 0x1aceac0] 8x8 transform intra:94.4% inter:18.8%
[libx264 @ 0x1aceac0] coded y,u,v intra: 2.8% 0.0% 0.0% inter: 0.2% 0.0% 0.0%
[libx264 @ 0x1aceac0] i16 v,h,dc,p:  5% 85% 10%  0%
[libx264 @ 0x1aceac0] i8 v,h,dc,ddl,ddr,vr,hd,vl,hu: 90%  9%  1%  0%  0%  0%  0%  0%  0%
[libx264 @ 0x1aceac0] i4 v,h,dc,ddl,ddr,vr,hd,vl,hu: 61% 15% 18%  1%  1%  1%  1%  0%  1%
[libx264 @ 0x1aceac0] Weighted P-Frames: Y:0.0% UV:0.0%
[libx264 @ 0x1aceac0] ref P L0: 97.3%  0.3%  1.9%  0.6%
[libx264 @ 0x1aceac0] ref B L0: 41.1% 58.9%
[libx264 @ 0x1aceac0] ref B L1: 99.9%  0.1%
[libx264 @ 0x1aceac0] kb/s:143.53
[31msynfig(9732) [05:34:24  IST] error: -3 layers not yet deleted![0m
```

I tried multiple times & each time synfig reports a different number of layers are not yet deleted

I'm not sure why the issue happens & how to resolve it (I did not use skeletons/anything else in my project)

I have attached the log
[synfig-rendering-error.log](https://github.com/synfig/synfig/files/2149319/synfig-rendering-error.log)

**Comment by thekalinga** [comment]
When I try to render the project using UI, it runs a different ffmpeg command than what it runs in the silent mode

```sh
# Underlying ffmpeg command ran by the above command
ffmpeg -f image2pipe -vcodec ppm -r 24.000000 -i pipe: -an -metadata title="Final Aggregate" -vcodec libx264 -b:v 2000k -y /home/user/Desktop/Presentation/Request-Per-Thread-End-To-End-Flow.mpg

# Underlying ffmpeg Command ran by UI
ffmpeg -f image2pipe -vcodec ppm -r 24.000000 -i pipe: -an -metadata title="Final Aggregate" -vcodec libx264 -b:v 2000k -tune fastdecode -pix_fmt yuv420p -qp 0 -y /home/user/Desktop/Presentation/Request-Per-Thread-End-To-End-Flow1.mkv
```

Also, how to make the terminal run the same command UI runs?

**Comment by ice0** [comment]
Hi, @thekalinga! Thank you for your response! This error is mostly for information purpose, your file must be rendered fine. Error is fixed, will be available in the next build.

**Comment by thekalinga** [comment]
@ice0 Thanks for your reply. Is there a place from where I can download this fixed synfig build?

**Comment by morevnaproject** [comment]
@thekalinga  The fix is now shipped with Synfig Studio 1.3.10 - https://www.synfig.org/2018/07/21/synfig-studio-1-3-10-released/

**Comment by thekalinga** [comment]
@morevnaproject Thanks for letting me know
